### PR TITLE
Fix diagram filenames, make them display properly

### PR DESCRIPTION
**Pull request by bact**
Remove spaces from filenames to make model images displayed properly.

### CODE REVIEW

**Review by bobmartin3000 — APPROVED** [review]
LGTM

File: docs/annexes/RDF-object-model-and-identifier-syntax.md [hunk 1]
```diff
@@ -6,19 +6,37 @@ TODO: update for SPDXv3
 
 SPDX ® Vocabulary Specification
 
-[SPDX 3.0 Model SHACL](https://spdx.github.io/spdx-3-model/model.ttl)
+[SPDX 3.0 Model SHACL](https://spdx.org/rdf/3.0.0/spdx-model.ttl)
```
Comment (kestewart) [inline docs/annexes/RDF-object-model-and-identifier-syntax.md:9]:
```suggestion
[SPDX 3.0 Model SHACL](https://spdx.org/rdf/3.0.1/spdx-model.ttl)
```

**Review by goneall — APPROVED** [review]
LGTM - Thanks for the quick updates.

**Comment by goneall** [comment]
@bact - can you rebase to the latest - merge is showing conflicts

**Comment by bact** [comment]
@goneall, again, I can't rebase it -- please use PR #1033 instead

**Comment by bact** [comment]
Conflicts resolved.

**Pull request merged**